### PR TITLE
Feature/1654 province filter

### PR DESCRIPTION
**Pull request by zamuzakki**
This is PR for #1654 
[Kazam_screencast_00058.webm](https://github.com/kartoza/sawps/assets/7352963/f2b76e4c-7245-4485-9a74-dfccdbd467b6)

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 31 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 30 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 22 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 19 sec

File: django_project/frontend/src/containers/MainPage/Metrics/index.scss [hunk 1]
```diff
@@ -337,3 +337,17 @@
         text-align: center;
     }
 }
+
+.trends-left-line-chart {
```
Comment (danangmassandy) [inline django_project/frontend/src/containers/MainPage/Metrics/index.scss:341]: perhaps, let's move this class to index.scss in Trends?

File: django_project/frontend/api_views/statistical.py [hunk 2]
```diff
@@ -125,6 +127,7 @@ def post(self, *args, **kwargs):
         trends = []
         with model_output.output_file.open('r') as json_file:
             json_dict = json.load(json_file)
+            print(PROPERTY_TREND in json_dict)
```
Comment (danangmassandy) [inline django_project/frontend/api_views/statistical.py:130]: please remove this print debug

**Review by danangmassandy — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 21 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 48 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 9 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 44 sec

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 26 sec

**Pull request merged**